src/pages/Home/components/MovieItem/ThreeDotMenuActions/ThreeDotMenuActions.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import MovieEdit from 'pages/Home/components/ModalWindow/MovieEdit/MovieEdit';
+import MovieDelete from 'pages/Home/components/ModalWindow/MovieDelete/MovieDelete';
+import CloseButton from 'pages/shared/CloseButton/CloseButton';
+import 'pages/Home/components/MovieItem/ThreeDotMenuActions/ThreeDotMenuActions.scss';
+
+class ThreeDotMenuActions extends React.Component {
+    state = {
+        editModalVisible: false,
+        deleteModalVisible: false,
+        threeDotMenuActionModalVisible: true
+    }
+
+    showEditModal = () => {
+        this.setState({editModalVisible: true});
+        this.setState({threeDotMenuActionModalVisible: false});
+    }
+
+    showDeleteModal = () => {
+        this.setState({deleteModalVisible: true});
+        this.setState({threeDotMenuActionModalVisible: false});
```

Comment:
you don't need to use few setStates here, one is enough:
```
this.setState({
   deleteModalVisible: true,
   threeDotMenuActionModalVisible: false
});
```

Reply:
Done.